### PR TITLE
Update build.sh to compile wps, mpas and rdas simultaneously

### PR DESCRIPTION
**Pull request by chunhuazhou**

### CODE REVIEW

**Review by guoqing-noaa — COMMENTED** [review]
Thanks, @chunhuazhou 

Could you update README.md as well?

**Comment by chunhuazhou** [comment]
> Thanks, @chunhuazhou
> 
> Could you update README.md as well?

I think you mean build and run info so I just updated it. Thanks!

File: docs/build_and_run.md [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
-./build.wps  | tee ./log.build.wps
-./build.mpas | tee ./log.build.mpas
-./build.rdas | tee ./log.build.rdas
 
+Upon completion of the build, users should see the following executables under `../exec`:
+```
+ atmosphere_model.x
+ init_atmosphere_model.x
+ mpasjedi_variational.x
+ ungrib.x
```
Comment (guoqing-noaa) [inline docs/build_and_run.md:21]: add `bufr2ioda.x` before `ungrib.x`

**Comment by guoqing-noaa** [comment]
> > Thanks, @chunhuazhou
> > Could you update README.md as well?
> 
> I think you mean build and run info so I just updated it. Thanks!

Yes, you are right, buid_and_run.md

File: docs/build_and_run.md (hunk 1 above)
Comment (chunhuazhou) [reply docs/build_and_run.md]: Added. Also reordered the executables according to the time they are being generated. Thanks!

**Review by guoqing-noaa — APPROVED** [review]
LGTM. Thanks, @chunhuazhou

**Pull request merged**